src/translations/sqlb_ar_SA.ts

```diff
@@ -6567,20 +7070,24 @@ Hold %3Shift and click to jump there</source>
     </message>
     <message>
         <source>Ctrl+H</source>
-        <translation></translation>
+        <translation type="vanished">Ctrl+H</translation>
```

Comment:
Shortcuts should not be translated.
See the [wiki](https://github.com/sqlitebrowser/sqlitebrowser/wiki/Translations#dont-translate-key-names-in-shortcuts) for further information.

**but this one is vanished entry so you can ignore this one.**